test/src/sdk-harness/test_projects/evm_ecr/src/main.sw

```diff
@@ -0,0 +1,42 @@
+script;
+
+use std::b512::B512;
+use std::address::Address;
+use std::vm::evm::ecr::ec_recover_address;
+use std::ecr::EcRecoverError;
+use std::assert::assert;
```

Comment:
Sort dependencies in Sway files alphabetically.

Reply:
Resolved : [b6c31ff](https://github.com/FuelLabs/sway/pull/1374/commits/b6c31ffd044cd0724506bfd423ee8b95edf00e9b)

I feel like if this is a hard requirement then `forc fmt` should be able to do it ? I was also concerned about import order, which in an ideal world is entirely superstitious but ...